server.js

```diff
@@ -15,27 +17,36 @@ app.get('/api/v1/tasks', (req, res) => {
   res.status(200).json({ tasks: app.locals.tasks });
 });
 
+app.get('/api/v1/savedtasks', (req, res) => {
+  res.status(200).json(app.locals.savedTasks);
+});
+
 app.get('/api/v1/tasks/:category', (req, res) => {
-  let categoryKeys = Object.keys(app.locals.tasks)
-  let arr = []
   let category = req.params.category
-  categoryKeys.forEach(key => {
-    app.locals.tasks[key].forEach((task) => {
-      let lowercase = task.category
-      let findWhiteSpace = lowercase.replace(/ /g, '')
-      lowercase = findWhiteSpace.toLowerCase()
-      if (lowercase === category) {
-        arr.push(task)
-      }
-    })
-  })
-  if (arr.length !== 0) {
-    res.status(200).json(arr)
+  if (req.params.category === 'mentalcare'){
+    category = 'mentalCare'
+  }
+  const foundTasks = app.locals.tasks[category.toLowerCase()]
+
+  if (foundTasks.length) {
+    res.status(200).json(foundTasks)
   } else {
     res.sendStatus(404)
   }
-})
+});
 
+app.post('/api/v1/savedtasks', (req, res) => {
+  app.locals.savedTasks.push(req.body);
+
+  res.status(201).json(app.locals.savedTasks);
```

Comment:
Curious because I don't know. Why do we respond to GET requests with 200 and post requests with 201?

Reply:
I didn't know honestly so I asked chatGPT and this was his response! 
"GET Request (200 - OK):
When a client sends a GET request to the server, it typically intends to retrieve data or resources. If the server successfully finds and returns the requested resource, it responds with a status code of 200 (OK). This indicates that the request was successful, and the response contains the requested data or resource.

POST Request (201 - Created):
When a client sends a POST request to the server, it usually intends to create a new resource on the server. For example, when submitting a form to add a new item to a database or creating a new user account. If the server successfully processes the request and creates the resource, it responds with a status code of 201 (Created). This status code informs the client that the request was successful and that a new resource has been created as a result."